File: app/src/main/java/com/bopr/android/smailer/ui/RulesActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * Conditions settings activity.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class RulesActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new RulesFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/RulesActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * Conditions settings activity.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class RulesActivity : BaseAppActivity() {

    override fun createFragment(): Fragment {
        return RulesFragment()
    }
}